initDatabase.py

```diff
@@ -1,26 +1,36 @@
 import csv
 import requests
 from requests.exceptions import HTTPError
-#By default all the courses will be posted into the database for testing purposes
-#Change the session wanted number to limit the number of created sessions
+
+# Ensure that the courses.csv file present
+# By default all the courses will be posted into the database for testing purposes
+# Change the session wanted number to limit the number of created sessions
 sessions_wanted = 56
 courseResponseIds = []
 postHeaders={
 }
 
-# Make sure sessions wanted is less than the number of classes in the csv which is 56
```

Comment:
Why not get the length of the CSV at runtime and set the max that way?

Reply:
Good point!